### PR TITLE
Feature/improved ux

### PR DESCRIPTION
**Pull request by pavankalyan-codes**
Let me test the changes first - @santoshimz will let you know once testing is done - you can review

### CODE REVIEW

File: src/components/Stats.js [hunk 1]
```diff
@@ -77,6 +90,19 @@ const Stats = ({ openStatsModal, setOpenStatsModal, statsObj, guessData }) => {
           );
         })}
       </div>
+      {isGameDone(gameStatus) && (
```
Comment (santoshimz) [inline src/components/Stats.js]: Nit: useMemo for this computation

Comment (pavankalyan-codes) [reply src/components/Stats.js]: done

**Comment by pavankalyan-codes** [comment]
@santoshimz, take a look now

**Comment by santoshimz** [comment]
We should either have all share actions inside modal or none. Since modal surfaces first, users will be inclined to use that and it will add additional friction to share on Twitter

**Comment by santoshimz** [comment]
Lgtm, thanks for the review fixes

**Pull request merged**